### PR TITLE
Stylus support

### PR DESCRIPTION
**Pull request by TobiasTimm**
Based on the help on slack. I've created a plugin for Stylus support.

### CODE REVIEW

File: packages/next-stylus/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        nextConfig.extractCSSPlugin || options.extractCSSPlugin
+
+      if (!extractCSSPlugin) {
+        extractCSSPlugin = new ExtractTextPlugin({
+          filename: 'static/style.css'
+        })
+        config.plugins.push(extractCSSPlugin)
+        options.extractCSSPlugin = extractCSSPlugin
+        if (!isServer) {
+          config = commonsChunkConfig(config, /\.(scss|sass)$/)
```
Comment (timneutkens) [inline packages/next-stylus/index.js]: This needs to be updated to `/\.styl/`

File: packages/next-stylus/readme.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+### With CSS modules and options
+
+You can also pass a list of options to the `css-loader` by passing an object called `cssLoaderOptions`.
+
+For instance, [to enable locally scoped CSS modules](https://github.com/css-modules/css-modules/blob/master/docs/local-scope.md#css-modules--local-scope), you can write:
+
+```js
+// next.config.js
+const withSass = require('@zeit/next-sass')
```
Comment (timneutkens) [inline packages/next-stylus/readme.md]: This mentions withSass instead of withStylus

File: packages/next-stylus/package.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@zeit/next-stylus",
+  "version": "0.1.1",
```
Comment (timneutkens) [inline packages/next-stylus/package.json]: 0.0.1

**Comment by TobiasTimm** [comment]
@timneutkens I don't wanna annoy you 😆 Would be cool if I can use the official plugin in my project soon

**Comment by timneutkens** [comment]
Published as next-stylus

**Comment by timneutkens** [comment]
@TobiasTimm lets also add it here: https://github.com/zeit/next.js#importing-css--sass--less-files

**Pull request merged**